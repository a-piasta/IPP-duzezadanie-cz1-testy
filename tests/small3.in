extendRoute;50;tmdlpltpqjkk
getRouteDescription;976
extendRoute;50;hxdjhiefvmusp
getRouteDescription;255
addRoad;wjynveringhpwsaovx;kmzyfqnooqmvygc;17;-15
addRoad;wjynveringhpwsaovx;kmzyfqnooqmvygc;12;12
getRouteDescription;651
repairRoad;pbggadottdh;rrojvbvhhugvzvfzkfvylinwbc;-15
extendRoute;50;rkdzmyjjbssmyzhmtbbrieb
addRoad;kfenjpwaqjlqbupuecoxxitonm;hbkgzpymdm;10;-5
getRouteDescription;211
addRoad;kfenjpwaqjlqbupuecoxxitonm;rrojvbvhhugvzvfzkfvylinwbc;27;1
getRouteDescription;258
repairRoad;rrojvbvhhugvzvfzkfvylinwbc;wjynveringhpwsaovx;0
extendRoute;50;tmdlpltpqjkk
removeRoad;kfenjpwaqjlqbupuecoxxitonm;rrojvbvhhugvzvfzkfvylinwbc
removeRoad;kmzyfqnooqmvygc;wjynveringhpwsaovx
removeRoad;wjynveringhpwsaovx;kmzyfqnooqmvygc
addRoad;rrojvbvhhugvzvfzkfvylinwbc;kmzyfqnooqmvygc;22;-4
addRoad;rrojvbvhhugvzvfzkfvylinwbc;tmdlpltpqjkk;4;-11
repairRoad;wjynveringhpwsaovx;pbggadottdh;-4
newRoute;651;rrojvbvhhugvzvfzkfvylinwbc;pbggadottdh
removeRoad;kmzyfqnooqmvygc;rrojvbvhhugvzvfzkfvylinwbc
extendRoute;651;aqenmkewxmfokyp
extendRoute;651;tmdlpltpqjkk
addRoad;tmdlpltpqjkk;rrojvbvhhugvzvfzkfvylinwbc;25;1
newRoute;1000;aqenmkewxmfokyp;wjynveringhpwsaovx
extendRoute;651;kmzyfqnooqmvygc
getRouteDescription;1000
addRoad;wjynveringhpwsaovx;kfenjpwaqjlqbupuecoxxitonm;24;1
addRoad;rrojvbvhhugvzvfzkfvylinwbc;aqenmkewxmfokyp;11;0
removeRoad;wjynveringhpwsaovx;kfenjpwaqjlqbupuecoxxitonm
removeRoad;wjynveringhpwsaovx;kmzyfqnooqmvygc
extendRoute;651;aqenmkewxmfokyp
repairRoad;hxdjhiefvmusp;aqenmkewxmfokyp;-7
addRoad;wjynveringhpwsaovx;pbggadottdh;23;15
newRoute;180;kfenjpwaqjlqbupuecoxxitonm;rkdzmyjjbssmyzhmtbbrieb
addRoad;rrojvbvhhugvzvfzkfvylinwbc;kfenjpwaqjlqbupuecoxxitonm;11;7
getRouteDescription;180
extendRoute;180;rrojvbvhhugvzvfzkfvylinwbc
repairRoad;kmzyfqnooqmvygc;kmzyfqnooqmvygc;14
addRoad;hbkgzpymdm;tmdlpltpqjkk;27;-10
removeRoad;kmzyfqnooqmvygc;wjynveringhpwsaovx
getRouteDescription;180
repairRoad;wjynveringhpwsaovx;rrojvbvhhugvzvfzkfvylinwbc;7
repairRoad;pbggadottdh;rkdzmyjjbssmyzhmtbbrieb;0
extendRoute;180;tmdlpltpqjkk
repairRoad;hbkgzpymdm;rkdzmyjjbssmyzhmtbbrieb;-13
extendRoute;180;rrojvbvhhugvzvfzkfvylinwbc
getRouteDescription;930
addRoad;pbggadottdh;pbggadottdh;28;-4
getRouteDescription;651
removeRoad;tmdlpltpqjkk;hbkgzpymdm
newRoute;786;hxdjhiefvmusp;tmdlpltpqjkk
extendRoute;651;rkdzmyjjbssmyzhmtbbrieb
extendRoute;651;aqenmkewxmfokyp
addRoad;hxdjhiefvmusp;tmdlpltpqjkk;20;13
extendRoute;651;kfenjpwaqjlqbupuecoxxitonm
addRoad;hbkgzpymdm;kmzyfqnooqmvygc;3;8
