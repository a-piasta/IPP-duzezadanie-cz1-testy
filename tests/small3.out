1: NIE
2: 
3: NIE
4: 
5: TAK
6: NIE
7: 
8: NIE
9: NIE
10: TAK
11: 
12: TAK
13: 
14: NIE
15: NIE
16: TAK
17: TAK
18: NIE
19: TAK
20: TAK
21: NIE
22: NIE
23: TAK
24: NIE
25: NIE
26: NIE
27: NIE
28: NIE
29: 
30: TAK
31: NIE
32: TAK
33: NIE
34: NIE
35: NIE
36: TAK
37: NIE
38: TAK
39: 
40: NIE
41: NIE
42: TAK
43: NIE
44: 
45: NIE
46: NIE
47: NIE
48: NIE
49: NIE
50: 
51: NIE
52: 
53: TAK
54: NIE
55: NIE
56: NIE
57: TAK
58: NIE
59: TAK
